### DISCUSSION TITLE
No such host

### DISCUSSION THREAD

**Discussion by r00tr4v3n** [aceberg/WatchYourLAN#127] [Q&A] [answered]
I getting this error when running the default docker compose file.

` ✘ wyl Error Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: no such hos...            0.0s 
Error response from daemon: Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: no such host
`

**Comment by aceberg** [comment, marked as answer]
Looks like your system can't find docker registry. If it's banned for some reason, you can try to download image from github: 
`ghcr.io/aceberg/watchyourlan:v2`
